python/samples/documentation_examples/ai_services.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import os
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+
+
+async def main():
+    # Initialize the kernel
+    kernel = sk.Kernel()
+
+    # Add the service to the kernel
+    # use_chat: True to use chat completion, False to use text completion
+    kernel = add_ai_service(kernel=kernel, use_chat=True)
```

Comment:
is it a good idea to rename this function to `add_service` so that it is the same when one uses it directly on the kernel?

Reply:
Sure, I think that makes sense and is clear.